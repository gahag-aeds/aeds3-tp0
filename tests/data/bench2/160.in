160 80
17 77 19 71 66 17 88 59 71 47 52 49 42 39 24 10 66 25 56 67 67 12 22 57 71 86 52 93 94 11 64 11 40 35 83 6 52 71 65 23 70 69 24 12 9 1 75 75 26 31 43 45 95 17 2 19 3 54 12 98 17 28 9 9 63 44 15 67 67 33 90 37 54 66 2 63 67 77 91 93 60 86 90 8 3 92 27 58 98 39 56 67 67 18 76 82 14 44 49 82 29 91 71 83 57 73 47 25 2 90 70 63 76 61 71 79 5 50 37 56 89 46 23 8 16 52 42 30 48 91 12 77 34 84 12 91 9 59 68 12 49 39 27 77 52 50 56 57 0 46 13 41 92 89 1 60 41 43 90 41 
Add 94 146
Min 41 93
Add 28 80
Min 91 143
Sub 61 113
Sum 35 87
Sub 14 66
Max 72 124
Add 50 102
Sum 70 122
Add 24 76
Sum 9 61
Sub 89 141
Min 89 141
Sub 63 115
Max 5 57
Sub 9 61
Min 98 150
Add 93 145
Max 60 112
Sub 85 137
Sum 51 103
Add 93 145
Sum 47 99
Sub 77 129
Sum 10 62
Sub 71 123
Min 59 111
Add 78 130
Min 12 64
Sub 44 96
Sum 103 155
Sub 52 104
Min 3 55
Add 86 138
Sum 44 96
Sub 76 128
Min 100 152
Add 8 60
Sum 18 70
Add 62 114
Max 32 84
Sub 66 118
Max 82 134
Add 52 104
Min 60 112
Add 86 138
Sum 100 152
Sub 50 102
Min 3 55
Sub 52 104
Max 28 80
Add 53 105
Max 38 90
Add 74 126
Max 22 74
Sub 86 138
Min 38 90
Sub 12 64
Min 57 109
Add 88 140
Min 94 146
Add 52 104
Max 32 84
Sub 28 80
Min 70 122
Sub 7 59
Sum 104 156
Sub 62 114
Max 14 66
Add 53 105
Min 55 107
Add 84 136
Sum 11 63
Add 68 120
Max 68 120
Sub 70 122
Max 46 98
Sub 60 112
Max 52 104
