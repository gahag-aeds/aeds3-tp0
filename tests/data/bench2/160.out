1
3
2457
97
3089
2395
1
97
0
99
2671
2618
2301
-1
1
2620
0
2419
0
2292
98
94
-2
2682
1
97
98
98
-2
-2
2
98
-1
2679
97
-1
2165
97
98
97
